modules/quickstart-uyuni/pages/install-uyuni-proxy.adoc

```diff
@@ -1,57 +1,94 @@
 
-The {productname} Proxy software is available from download.opensuse.org, and you can use zypper to retrieve the software and install it.
+== Mirror {productname} Proxy software
 
+The {productname} Proxy software is available from https://download.opensuse.org.
+You can synchronize the proxy software to your {productname} Server.
+This process is also known as mirroring.
 
 
-.Procedure: Installing openSUSE Leap with Uyuni Proxy
-. Install openSUSE Leap as the base system, and ensure all package updates have been applied.
-. Configure a resolvable fully qualified domain name (FQDN) with {yast} by navigating to menu:System[Network Settings > Hostname/DNS].
-. At the command prompt, as root, add the repository for installing the {productname} Server software:
-// variable assignment to avoid overlong lines
-// https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/
-+
-----
-repo=repositories/systemsmanagement:/
-repo=${repo}Uyuni:/Stable/images/repo/Uyuni-Proxy-POOL-x86_64-Media1/
-zypper ar https://download.opensuse.org/$repo uyuni-proxy-stable
-----
-. Refresh metadata from the repositories:
+
+.Procedure: Mirroring {productname} Proxy software
+. On the {productname} Server, create openSUSE Leap and the {productname} Proxy channels with the [command]``spacewalk-common-channels`` command.
+  [command]``spacewalk-common-channels`` is part of the [package]``spacewalk-utils`` package:
 +
 ----
-zypper ref
+spacewalk-common-channels \
+opensuse_leap15_2 \
+opensuse_leap15_2-non-oss \
+opensuse_leap15_2-non-oss-updates \
+opensuse_leap15_2-updates \
+opensuse_leap15_2-uyuni-client \
+uyuni-proxy-stable-leap-152
 ----
-. Install the pattern for the {productname} Proxy:
 +
+Instead of the [systemitem]``uyuni-proxy-stable-leap-152`` version you can also try the latest development version, called [systemitem]``uyuni-proxy-devel-leap``.
+For more information, see xref:client-configuration:clients-opensuse.adoc[].
+
+
+
+== Register the openSUSE Leap system
+
+Begin by installing openSUSE Leap on a physical or virtual machine.
+To ensure that the proxy is accessible across the network, you must have a resolvable fully-qualified domain name (FQDN) on the openSUSE Leap system before you begin the installation.
+You can configure an FQDN with {yast} by navigating to menu:System[Network Settings > Hostname/DNS].
+
+Prepare the proxy installation settings on the server and register the openSUSE Leap system as a client.
+
+
+
+.Procedure: Registering the openSUSE Leap system
+. On the {productname} Server, create an activation key with openSUSE Leap as a base channel and the proxy and the other channels as child channels.
+  For more information about activation keys, see xref:client-configuration:activation-keys.adoc[].
+. Modify a bootstrap script for the proxy.
+  Add the GPG key to the [systemitem]``ORG_GPG_KEY=`` parameter.
```

Comment:
```suggestion
  If required, add your GPG key to the [systemitem]``ORG_GPG_KEY=`` parameter.
```

Reply:
I think I understand now what GPG key you are talking about. The one for Uyuni, I guess, as we suggest for regular clients? I so, this should be specified here.

Reply:
Can we agree on the exact wording?  For me it worked, if I'd use one of those keys provided on the server in the bootstrap directory.  Thus at least by default, you can use the "official" keys.  I'm not sure whether we should this key "your GPG key" here?

I think, by default, the key is always required.  Maybe, it is not needed if you use some kind of cloned channels.

My original wording was probably to short, but I think we must say something along these lines: "Adding the {productname} provided GPG key is mandatory." I could be wrong, though.  @juliogonzalez and @Loquacity can you please help again?

Reply:
@keichwa AFAIK you need to manually add the Uyuni key (which is not *your* key) to ``ORG_GPG_KEY=`` or am I missing something?

Otherwise when you bootstrap the proxy that will work. But if you need to install any other packages, or update, it won't work.

So shouldn't we remove "your GPG key" from here, and just tell the user:

>  Add the GPG key for Uyuni to the [systemitem]``ORG_GPG_KEY=`` parameter like this: ``ORG_GPG_KEY=whatever``

whatever, because I don't remember what the filename is. But it's easy if you look at the server at `/srv/www/htdocs/pub/` (or otherwise via web https://hostname/pub/

Reply:
Updated

Reply:
![image](https://user-images.githubusercontent.com/3914967/107315226-fbd29580-6ae1-11eb-8fd0-dea04f724099.png)